public/app/plugins/panel/geomap/components/DataHoverTabs.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { Dispatch, SetStateAction } from 'react';
+import { Tab, TabsBar } from '@grafana/ui';
+
+import { GeomapLayerHover } from '../event';
+
+type Props = {
+  layers?: GeomapLayerHover[];
```

Comment:
```suggestion
  layers: GeomapLayerHover[];
```
and avoid null check?